release/DOCKERFILE

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu
```

Comment:
Should we call this file Dockerfile to make it consistent with the names in other folders?

Reply:
Let's also pin the dependency to `FROM ubuntu:bionic@sha256:eb1392bbdde63147bc2b4ff1a4053dcfe6d15e4dfd3cce29e9b9f52a4f88bc74`

Reply:
Updated